commandServer/routes.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+
+class BaseModel(object):
+
+    def get(self, request, params=None):
+        return json.dumps([]).encode('utf_8')
```

Comment:
all that json serialization must be only part of BaseModel. We need something like dispatch() method in Django to process request and response and have more control over it